internal/config/client_configuration.go

```diff
@@ -49,7 +50,7 @@ func (s SatelliteConfiguration) Merge(config Mergable) Mergable {
 func DefaultSatelliteConfiguration() SatelliteConfiguration {
 	return SatelliteConfiguration{
 		Groundstation: Groundstation{"localhost", 8080},
-		Satellite:     Satellite{"/tmp/satellite", 60, 2},
+		Satellite:     Satellite{"/tmp/satellite", 60, 2, false},
```

Comment:
NIT: At this point, this should use field names, as it's not at all obvious what's going on here

Reply:
Fixed in 52ff904